squad-server/log-parser/player-suicide.js

```diff
@@ -0,0 +1,17 @@
+export default {
+  regex: /^\[([0-9.:-]+)]\[([ 0-9]*)]LogSquad: Warning: Suicide (.+)/,
```

Comment:
Is this still valid with the current version of squad?

Reply:
> Is this still valid with the current version of squad?

I don't have any clue as I didn't touch squad for ages now :/